### PR TITLE
Nalindas9/faulty parts new

### PR DESCRIPTION
**Pull request by nalindas9**
Handling faulty parts

### CODE REVIEW

File: .vscode/c_cpp_properties.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+    "configurations": [
+        {
+            "browse": {
```
Comment (SaumilShah66) [inline .vscode/c_cpp_properties.json]: bro...remove this file from github.

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+    "python.autoComplete.extraPaths": [
+        "/home/varun/ariac_ws/devel/lib/python2.7/dist-packages",
```
Comment (SaumilShah66) [inline .vscode/settings.json]: This file too. These files are for IDE.

**Review by SaumilShah66 — CHANGES_REQUESTED** [review]
remove vscode files

File: .vscode/c_cpp_properties.json (hunk 1 above)
Comment (nalindas9) [reply .vscode/c_cpp_properties.json]: Done

File: .vscode/settings.json (hunk 2 above)
Comment (nalindas9) [reply .vscode/settings.json]: Done

**Pull request merged**